common/components/BalanceSidebar/Promos.jsx

```diff
@@ -0,0 +1,104 @@
+// @flow
+import './Promos.scss';
+import React from 'react';
+import { Link } from 'react-router';
+
+const promos = [
+  {
+    color: '#6e9a3e',
+    href:
+      'https://myetherwallet.groovehq.com/knowledge_base/topics/protecting-yourself-and-your-funds',
+    isExternal: true,
+    texts: [<h6 key="1">Learn more about protecting your funds.</h6>],
+    images: [
+      require('assets/images/logo-ledger.svg'),
+      require('assets/images/logo-trezor.svg')
+    ]
+  },
+  {
+    color: '#2b71b1',
+    href:
+      'https://buy.coinbase.com?code=a6e1bd98-6464-5552-84dd-b27f0388ac7d&address=0xA7DeFf12461661212734dB35AdE9aE7d987D648c&crypto_currency=ETH&currency=USD',
+    isExternal: true,
+    texts: [
+      <p key="1">It’s now easier to get more ETH</p>,
+      <h5 key="2">Buy ETH with USD</h5>
```

Comment:
When i was working on ENS general info list, I found it pretty weird having to manually assign keys in an array literal, maybe we should eventually figure out a solution around this in the future